### PR TITLE
Various tutorial improvements

### PR DESCRIPTION
**Pull request by jwmueller**
Another round of many tutorial improvements such as:
- Not having weird reshape in forward pass of MNIST ConvNet (also verified this works with the LearningWithNoisyLabels).
- Removing seemingly unnecessary TF logging suppression line in audio tutorial
- Moving the hidden tests to the end of tutorials and add tests to ensure LNL improves the model accuracy.
- Language and code improvements throughout tutorials.

Apologies for making PR from branch, my fork's master branch is occupied by open PR in limbo (was made from plane with bad internet)

### CODE REVIEW

File: docs/source/tutorials/audio.ipynb [hunk 1]
```diff
@@ -435,7 +436,7 @@
     "\n",
     "model = LogisticRegression(C=0.01, max_iter=1000, tol=1e-1, random_state=SEED)\n",
     "\n",
-    "# Generate cross-validated predicted probabilities for each datapoint\n",
+    "num_crossval_folds = 5  # can decrease this value to reduce runtime, or increase it to get better results\n",
     "cv_pred_probs = cross_val_predict(\n",
     "    estimator=model, X=embeddings_array, y=df.label.values, cv=5, method=\"predict_proba\"\n",
     ")"
```
Comment (weijinglok) [inline docs/source/tutorials/audio.ipynb:442]: `num_crossval_folds` seems to be defined here, but not directly used in `cross_val_predict`

**Comment by weijinglok** [comment]
@jwmueller, text and tabular tutorial have the additional check whether lnl accuracy > og accuracy, but this seems to be missing for audio and image tutorial. Was this intentional?

PS: thanks for vetting the tutorials!

**Review by weijinglok — CHANGES_REQUESTED** [review]
Overall looking good! just a couple small comments to be addressed!

File: docs/source/tutorials/audio.ipynb (hunk 1 above)
Comment (jwmueller) [reply docs/source/tutorials/audio.ipynb:442]: good catch!

Comment (jwmueller) [reply docs/source/tutorials/audio.ipynb]: fixed it

**Review by weijinglok — APPROVED** [review]
LGTM!

**Pull request merged**